components/prism-neon.js

```diff
@@ -0,0 +1,39 @@
+Prism.languages.neon = {
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'datetime': {
+		pattern: /(^|[[{(=,\s])\d\d\d\d-\d\d?-\d\d?(?:(?:[Tt]| +)\d\d?:\d\d:\d\d(?:\.\d*)? *(?:Z|[-+]\d\d?(?::?\d\d)?)?)?(?=$|[\]}),\s])/,
+		lookbehind: true,
+		alias: 'number'
+	},
+	'key': {
+		pattern: /(^|[[{(,\s])[^,:=[\]{}()'"\s]+(?=\s*:(?:$|[\]}),\s])|\s*=)/,
+		lookbehind: true,
+		alias: 'atrule'
+	},
+	'number': {
+		pattern: /(^|[[{(=,\s])[+-]?(?:0x[\da-fA-F]+|0o[0-7]+|0b[01]+|(?:\d+\.?\d*|\.?\d+)(?:[eE][+-]?\d+)?)(?=$|[\]}),\s])/,
+		lookbehind: true
+	},
+	'boolean': {
+		pattern: /(^|[[{(=,\s])(?:true|false|yes|no)(?=$|[\]}),\s])/i,
+		lookbehind: true
+	},
+	'null': {
+		pattern: /(^|[[{(=,\s])(?:null)(?=$|[\]}),\s])/i,
+		lookbehind: true
+	},
+	'string': {
+		pattern: /(^|[[{(=,\s])(?:'''\r?\n(?:(?:[^\r\n]|\r?\n(?![\t\ ]*'''))*\r?\n)?[\t\ ]*'''|"""\r?\n(?:(?:[^\r\n]|\r?\n(?![\t\ ]*"""))*\r?\n)?[\t\ ]*"""|'[^'\n]*'|"(?:\\.|[^"\n])*")(?=$|[\]}),\s])/m,
```

Comment:
1. Why the `m` flag? All other patterns don't seem to need it.
1. The pattern will be shorter if we combine `'''\r?\n(?:(?:[^\r\n]|\r?\n(?![\t\ ]*'''))*\r?\n)?[\t\ ]*'''` and `"""\r?\n(?:(?:[^\r\n]|\r?\n(?![\t\ ]*"""))*\r?\n)?[\t\ ]*"""` using backreferences.
1. The single line strings are missing a `\r` in their character set.

```suggestion
		pattern: /(^|[[{(=,\s])(?:('''|""")\r?\n(?:(?:[^\r\n]|\r?\n(?![\t ]*\2))*\r?\n)?[\t ]*\2|'[^'\r\n]*'|"(?:\\.|[^"\r\n])*")(?=$|[\]}),\s])/,
```

Reply:
Yeah, I agree with everything, thanks.